src/components/content/order/ServiceList.tsx

```diff
@@ -281,9 +311,21 @@ function ServiceList(): JSX.Element {
         getServices();
     }
 
+    const handleOk = () => {
+        setIsModalOpen(false);
+    };
+
+    const handleCancel = () => {
+        setIsModalOpen(false);
+    };
+
     return (
         <div className={'services-content'}>
             {tip}
+
```

Comment:
we can avoid displaying OK and Cancel buttons. We don't need them. 
We can just have the close button (x).